src/main/java/org/folio/repository/CustomFieldsConstants.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.repository;
+
+public class CustomFieldsConstants {
+
+  public static final String CUSTOM_FIELDS_TABLE = "custom_fields";
+  public static final String JSONB_COLUMN = "jsonb";
+
+  static final String COUNT_CUSTOM_FIELDS_BY_REF_ID = "SELECT COUNT(*) as count from %s WHERE "+ JSONB_COLUMN +"->>'refId' LIKE ?";
```

Comment:
I think we decided to change this to id from refId - @andrii-paias - is that right?

Reply:
changed